cargo-process.el

```diff
@@ -283,6 +283,15 @@ Cargo: Build and execute src/main.rs."
   (cargo-process--start "Run" "cargo run"))
 
 ;;;###autoload
+(defun cargo-process-run-bin (command)
+  "Run the Cargo run command --bin <name>.
+With the prefix argument, modify the command's invocation.
+Cargo: Build and execute with --example <name>."
```

Comment:
This line is incorrect, it should be:
Cargo: Run the binary target of the local package